src/applications/disability-benefits/all-claims/components/Autocomplete.jsx

```diff
@@ -113,14 +121,30 @@ const Autocomplete = ({
         selectResult(results[activeIndex]);
       } else if (e.key === 'Escape') {
         closeList();
+      } else {
+        focusOnInput();
       }
     }
   };
 
-  const handleBlur = () => setTimeout(closeList, 200); // Enables clicking option from list
+  useEffect(() => {
+    const handleClickOutside = event => {
+      if (
+        containerRef.current &&
+        !containerRef.current.contains(event.target)
+      ) {
+        closeList();
+      }
+    };
+
+    document.addEventListener('mousedown', handleClickOutside);
```

Comment:
We'll want to switch this to `mouseup` so that users can change their mind about the action / cancel it. Check out this doc for more info: https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html#dfn-down-event

Reply:
Doh sorry I got this one wrong, it would be mouse up when selecting the item. This is related to `handleClickOutside.` Let's switch it back to mouse down 😅  Sorry!